.ci/install.sh

```diff
@@ -2,10 +2,19 @@
 
 set -e
 
-INSTALL_DIR="$HOME/.fnm"
 RELEASE="latest"
 OS="$(uname -s)"
 
+if [ -d "$HOME/.fnm" ]; then
+  INSTALL_DIR="$HOME/.fnm"
+elif [ ! -z $XDG_DATA_HOME ]; then
+  INSTALL_DIR="$XDG_DATA_HOME/fnm"
+elif [ "$OS" = "Darwin" ]; then
+  INSTALL_DIR="$HOME/Library/Application\ Support/fnm"
```

Comment:
You need to remove the extra `\`, because double quotes are already used. Otherwise the path is wrong.

Reply:
Thanks for pointing out my mistake.